### PR TITLE
chore: Code cleanup group creation

### PR DESCRIPTION
**Pull request by gregor**

### CODE REVIEW

File: app/script/components/userList.js [hunk 1]
```diff
@@ -75,9 +75,9 @@ z.components.UserList = class UserList {
           const trimmedQuery = this.filter().trim();
           const isHandle = trimmedQuery.startsWith('@') && z.user.UserHandleGenerator.validate_handle(normalizedQuery);
           const excludedEmojis = Array.from(normalizedQuery).filter(char => EMOJI_UNICODE_RANGES.includes(char));
-          return this.userEntities().filter(userEntity =>
-            userEntity.matches(normalizedQuery, isHandle, excludedEmojis)
-          );
+          return this.userEntities().filter(userEntity => {
+            return userEntity.matches(normalizedQuery, isHandle, excludedEmojis);
```
Comment (AndyLnd) [inline app/script/components/userList.js:79]: Why more overhead?

File: app/script/view_model/content/GroupCreationViewModel.js [hunk 2]
```diff
@@ -54,11 +54,9 @@ z.ViewModel.content.GroupCreationViewModel = class GroupCreationViewModel {
 
     this.activateNext = ko.pureComputed(() => this.nameInput().length);
     this.contacts = ko.pureComputed(() => {
-      if (this.showContacts()) {
-        return this.teamRepository.isTeam() ? this.teamRepository.teamUsers() : this.userRepository.connected_users();
-      }
-
-      return [];
+      return this.showContacts()
+        ? this.teamRepository.isTeam() ? this.teamRepository.teamUsers() : this.userRepository.connected_users()
```
Comment (AndyLnd) [inline app/script/view_model/content/GroupCreationViewModel.js]: IMO that is not really more readable.

File: app/script/components/userList.js (hunk 1 above)
Comment (gregor) [reply app/script/components/userList.js:79]: It's not overhhead. If we do not do an explicit return in this case you need to read two lines to see that something is returned due to it being split into multiple lines by prettier. With this you immediately see what's happening.

File: app/script/view_model/content/GroupCreationViewModel.js (hunk 2 above)
Comment (lipis) [reply app/script/view_model/content/GroupCreationViewModel.js]: I agree.. too many `?`, `:`

**Pull request merged**